src/actions/Action.ts

```diff
@@ -10,13 +10,9 @@ class Action implements IAction {
 
   getSortedParams(): Param[] {
     return (Array.from(this.params.values()) as Param[]).sort((param1, param2) => {
-      if (param1.key < param2.key) {
-        return -1;
-      } else if (param1.key > param2.key) {
-        return 1;
-      } else {
-        return 0;
-      }
+      // No need to check for equality because params is a map.
+      // Also, returning 0 will be unreachable code that will break code coverage.
+      return (param1.key > param2.key) ? 1 : -1;
```

Comment:
The return 0 is non reacahable, but it's part of the spec of sort (It can be confusing reading this sort without an equality condition).

I don't think this function should assume the type of params(now it's a map, but it could change)

If we want we can strictly check that params is a map by checking that `params` is instanceof map.
I agree that the equality condition is never reached in the current implementation.

Reply:
I suggest that if/when this becomes a util, it will be called sortMapByKeys or something similar. I've added the comments that explain this, so that any future dev can easily understand and make any change.

Reply:
My only concern is that comments lie (eventually, if the implementation will change)
I still prefer to have either a strict check or just include the equality operation.

It shouldn't break the code coverage, but if we're really concerned we could omit that logical branch from code coverage 

https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md

Reply:
Moved to sortMapByKey